frontend/apps/erd-sample/schema.rb

```diff
@@ -0,0 +1 @@
+../../packages/cli/fixtures/input.schema.rb
```

Comment:
Now using a symlink.

- During `pnpm dev`, the small file `packages/cli/fixtures/input.schema.rb` is still used.
- For PR previews, it is overridden by the larger file `mastodon/mastodon/db/schema.rb`.

**NOTE**: This update is a temporary measure until the CLI `erd build` supports the `--input [URL]` option.